pages/events/[id]/index.js

```diff
@@ -36,9 +36,19 @@ export default function EventDetails() {
     error,
   } = useSWR(id ? `/api/events/${id}` : null);
 
+  const eventBudget = event?.budget ?? "No Budget";
+
+  const totalExpenses =
+    event?.food +
+      event?.accomodation +
+      event?.transport +
+      event?.gifts +
+      event?.otherExpenses ?? "Amount Null";
```

Comment:
I think it would be better to use something like  `Array.reduce` to calculate the sum.